packages/cli/src/oclif/commands/pull.js

```diff
@@ -0,0 +1,43 @@
+const ZapierBaseCommand = require('../ZapierBaseCommand');
+const { downloadSourceZip } = require('../../utils/api');
+const { copyDir, makeTempDir, removeDir, validateFileExists, deleteUnmatchedFiles,
+} = require('../../utils/files');
+const AdmZip = require('adm-zip');
+const colors = require('colors/safe');
+const constants = require('../../constants');
+const debug = require('debug')('zapier:pull');
+const ignore = require('ignore');
+const { getGitIgnorePatterns } = require('../../utils/pull');
+
+class PullCommand extends ZapierBaseCommand {
+  async perform() {
+    if (
+      !await this.confirm(colors.yellow('This will overwrite your local integration files with the latest version. Continue?'))
+    ) {
+      this.exit();
+    }
+
+    await downloadSourceZip();
+    validateFileExists(constants.SOURCE_PATH);
+
+    const tmpDir = makeTempDir();
+    debug('Using temp directory for source unzip: ', tmpDir);
+
+    const zip = new AdmZip(constants.SOURCE_PATH)
+    zip.extractAllTo(tmpDir, true)
+
+    const gitIgnorePatterns = await getGitIgnorePatterns('example-target');
```

Comment:
Everywhere you see 'example-target' is just used for testing. We'll remove this prior to merge.